pkg/prompt/prompt.go

```diff
@@ -72,7 +72,7 @@ func (prompt *Prompt) Select(msg string, choices interface{}) (int, error) {
 	choice := prompt.Input(fmt.Sprintf("(%d-%d): ", 1, choicesLen))
 	i, err := strconv.Atoi(choice)
 	if err != nil {
-		return 0, err
+		return 0, errors.New("unable to parse selected input")
```

Comment:
```suggestion
		return 0, fmt.Errorf("unable to parse selected input: %s", err)
```

I'd like to have more context in errors